### PR TITLE
Create CHANGES-guidelines.md

### PR DESCRIPTION
**Pull request by jongiddy**
Copy of https://github.com/dcos/dcos/pull/7226 with suggested changes

### CODE REVIEW

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jongiddy** [comment]
@mesosphere-mergebot request-review @DominikDary

File: CHANGES-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* Entry with no newline characters. (DCOS_OSS_JIRA)
+<empty-line>
+* Another entry with no newline characters. (DCOS_OSS_JIRA_2)
+```
+
+Add new entries to the top of the existing content within a section. Each entry should be a proper English sentence, reflecting the change in easily comprehensible terms using the past tense. A JIRA ticket ID, if applicable, should be provided side-by-side with the entry as shown above.
+
+If the entry describes a bug fix in response to a customer operations (COPS) ticket then please also add the corresponding COPS ticket ID.
+
+### What is (not) a user-facing change?
```
Comment (jongiddy) [inline CHANGES-guidelines.md:33]: I've included a summary in the "How to choose..." section. Do we still need this section?

Comment (DominikDary) [reply CHANGES-guidelines.md:33]: no, let's remove the section

**Comment by DominikDary** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**